test/utils/update/review_trusts.spec.ts

```diff
@@ -50,27 +61,67 @@ describe('Manage trusts - review trusts utils tests', () => {
     });
   });
 
-  describe('putTrustInReview', () => {
+  describe('setupNextTrustForReview', () => {
     test.each([
       [ 'no update in model', undefined ],
       [ 'no review trusts', { review_trusts: undefined } ],
       [ 'review trusts is empty', { review_trusts: [] } ],
     ])('will return false when there is %s', (_, update) => {
       const appData = { update };
 
-      const result = putTrustInReview(appData);
+      const result = setupNextTrustForReview(appData);
 
       expect(result).toBe(false);
     });
 
-    test('will return true when there is a trust, and set its in_review status to true', () => {
-      const trust = { review_status: { in_review: false } } as Trust;
+    test('will return true when there is a trust, and setup its review_status', () => {
+      const trust = { trust_name: 'Trust 1' } as Trust;
       const appData = { update: { review_trusts: [trust] } };
 
-      const result = putTrustInReview(appData);
+      const result = setupNextTrustForReview(appData);
 
       expect(result).toBe(true);
-      expect(trust.review_status?.in_review).toBe(true);
+      expect(trust.review_status).toEqual({
+        in_review: false,
```

Comment:
need to align with latest model change reviewed_trust_details: false
can do that as follow on but wanted to point it out.